core/src/main/scala/kafka/server/AbstractFetcherThread.scala

```diff
@@ -184,7 +184,7 @@ abstract class AbstractFetcherThread(name: String,
     }
     catch {
       case e: KafkaStorageException =>
-        info(s"Failed to truncate $topicPartition " + s"at offset ${truncationState.offset}", e)
+        info(s"Failed to truncate $topicPartition at offset ${truncationState.offset}", e)
```

Comment:
Not from this patch, but it's a little inconsistent that we report this at INFO level while we report the same error in the fetch handling at ERROR. I think ERROR seems more accurate, so perhaps we can change this?

Reply:
Yeah, changed it to `ERROR`.